test/functionalTest/cases/3004_v2_reg_create/v2_reg_create_errors.test

```diff
@@ -77,6 +77,7 @@ brokerStart CB 0-255 IPv4 -statCounters
 # 46. supportedForwardingMode set to 'none'
 # 47. supportedForwardingMode set to 'query'
 # 48. supportedForwardingMode set to 'update'
+# 49. dataProvided:entities:entity:idPattern too complex
 #
```

Comment:
I'd suggest to add a #50 for typePattern

Reply:
Will do

Reply:
Fixed in 3dc4bcf83f00f7937ce79cacd25417575f5beaed